### ISSUE TITLE
schemas ingested via repositoryFactory do not appear in the playground schemas tab

### ISSUE THREAD

**Issue by squidsoup** [cube-js/cube.js#3908]
**Describe the bug**
When implementing a custom `repositoryFactory` (e.g. my [s3 implementation](https://gist.github.com/squidsoup/e1b72eee20bfe92484f7f4b98db518da)), files ingested by cubejs do not appear under the schemas tab in the playground, although `measures` and `dimensions` appear as expected.

**To Reproduce**
1. Implement a custom `repositoryFactory` in `cube.js`
2. visit `/#/schema` in the playground
3. note that files are not listed there

**Expected behavior**
A list of files (e.g. files ingested via s3) appears in the list

**Version:**
v0.28.57

**Comment by ovr** [comment]
Hello @squidsoup ,

Can you verify that you've implemented the `dataSchemaFiles` method for your own repository class?

Thanks

**Comment by squidsoup** [comment]
@ovr yes, you can see that here https://gist.github.com/squidsoup/e1b72eee20bfe92484f7f4b98db518da#file-s3-repofactory-js-L61

**Comment by paveltiunov** [comment]
@squidsoup I can confirm this isn't a supported use case. Please feel free to provide PR for it.
